src/platform_impl/windows/event_loop.rs

```diff
@@ -360,17 +398,15 @@ fn wait_thread(parent_thread_id: DWORD, msg_window_id: HWND) {
             msg = mem::zeroed();
 
             if wait_until_opt.is_some() {
-                if 0 != winuser::PeekMessageW(&mut msg, ptr::null_mut(), 0, 0, winuser::PM_REMOVE) {
-                    winuser::TranslateMessage(&msg);
-                    winuser::DispatchMessageW(&msg);
+                if PeekMessageW(&mut msg, 0, 0, 0, PM_REMOVE) == true.into() {
+                    TranslateMessage(&msg);
+                    DispatchMessageW(&msg);
                 }
+            } else if !GetMessageW(&mut msg, 0, 0, 0) == true.into() {
```

Comment:
Semantic change (positive comparison to `true` instead of negative comparison to `false`)

Also, this one is a bit confusing to read